test/extended/templates/templateinstance_readiness.go

```diff
@@ -118,6 +118,7 @@ var _ = g.Describe("[Conformance][templates] templateinstance readiness test", f
 		g.AfterEach(func() {
 			if g.CurrentGinkgoTestDescription().Failed {
 				exutil.DumpPodStates(cli)
+				exutil.DumpConfigMapStates(cli)
 				exutil.DumpPodLogsStartingWith("", cli)
```

Comment:
is it time to make this a utility so we don't have to touch every file whenever we want to update what we dump on failure?